### PR TITLE
Documents related to the association

### PR DESCRIPTION
**Pull request by Defman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,19 +10,18 @@ The budget is public and can be found  [here](./budget.md).
...
-- The treasure _____
-- _____
-- _____
-
-### Substitutes
-- _____
-- _____
+- The chairman Caelum van Ispelen
+- The deputy chairman Jacob Emil Ulvedal Rosborg
+- The treasure Niklas Schurwanz
```
Comment (Schuwi) [inline README.md]: missing an 'r' here

File: README.md [hunk 2]
```diff
@@ -10,19 +10,18 @@ The budget is public and can be found  [here](./budget.md).
...
+- Pau Machetti
+- Duy Nguyen
+- Miro Andrin Debrunner
+- Amber Kowalski
+- Gijsde Jong
```
Comment (Schuwi) [inline README.md]: I think it's "Gijs de Jong"

File: README.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Feather Minecraft Association
 The purpose of the Feather Minecraft Association is to facilitate the development of Feather, a Minecraft server implementation written in Rust, and related services like the plugin registry. Membership fee and donations go towards this goal and are used to maintain infrastructures like servers and domain name.   
 
-The association is lead by the elected Board of Directors, they are responsible for daily administrative work. They are elected once a year by the present members at the General Assembly which is held once a year and by the end of ____. 
+The association is lead by the elected Board of Directors, they are responsible for daily administrative work. They are elected once a year by the present members at the General Assembly which is held once a year and by the end of April.
```
Comment (Schuwi) [inline README.md]: "once a year, by the end of April" is better imo

File: README.md [hunk 4]
```diff
@@ -10,19 +10,18 @@ The budget is public and can be found  [here](./budget.md).
...
+- Gijsde Jong
 
 ## Contact details
 ### Email
 - You can contact the Board of Directors at board@feathermc.org
 
 ### CVR
-The Association is registered under the name "Feather Minecraft Association" in Denmark and has been assigned the CVR: [_____](https://datacvr.virk.dk/data/index.php?enhedstype=virksomhed&id=26858461&q=visenhed&language=en-gb).
\ No newline at end of file
+The Association is registered under the name "Feather Minecraft Association" in Denmark and has been assigned the CVR: [42166553](https://datacvr.virk.dk/data/index.php?enhedstype=virksomhed&id=42166553&q=visenhed&language=en-gb).
```
Comment (Schuwi) [inline README.md]: it's registered as "FeatherMC Association"

File: README.md (hunk 2 above)
Comment (oxkitsune) [reply README.md]: Yes it is 😂

**Comment by Schuwi** [comment]
@caelunshun can we get a merge on this please

**Pull request merged**